zigzag/zigzag_test_log.py

```diff
@@ -94,6 +95,18 @@ def qtest_testcase_id(self):
             self._lookup_ids()
         return self._qtest_testcase_id
 
+    @property
+    def job_config_attributes(self):
```

Comment:
Agree this is test case information.

Reply:
That makes sense.

Reply:
Can we pick another name for this property and the associated lookup?  Since this basically pytest expressing parameterization 
https://www.google.com/url?sa=t&rct=j&q=&esrc=s&source=web&cd=1&cad=rja&uact=8&ved=2ahUKEwi3puXB1ZjfAhXDjlQKHT-3CG4QFjAAegQIBBAB&url=http%3A%2F%2Fdoc.pytest.org%2Fen%2Flatest%2Fexample%2Fparametrize.html&usg=AOvVaw0lfyN3JCxDWmEZUtySjBrw

How about `test_execution_ parameters`?

Reply:
Test_execution_parameters sounds good.  I've kinda been puzzling over what to name it all along.